.github/actions/is-cache-disabled/action.yml

```diff
@@ -0,0 +1,27 @@
+---
+name: Is cache disabled
+
+description: |
+  Checks if current PR contains a label 'ci:no-cache'
+  This information can be used to skip cache restoration
+  in the setup-maven-cache and setup-yarn-cache to test
+  workflows from scratch
+
+inputs: {}
+
+outputs:
+  is-cache-disabled:
+    description: Output whether the current PR has a label 'ci:no-cache' to disable cache restoration
+    value: ${{ steps.detect-cache-disabled.outputs.is-cache-disabled == 'true' }}
+
+runs:
+  using: composite
+  steps:
+
+  - id: detect-cache-disabled
+    env:
+      IS_CACHE_DISABLED: ${{ contains(github.event.pull_request.labels.*.name, 'ci:no-cache') }}
```

Comment:
❓ Does this work as expected* when not on a PR?

*I'd expect that e.g. `on: push` to main or stable branches or `on: schedule` etc the cache restoration is **enabled** (should be the default) and this action should return `is-cache-disabled == false`.

Reply:
I've updated my poc and created two new workflows, one with `on: push` events (Push workflow) and one with `on: workflow_dispatch` (Dispatch workflow) (I will assume it's the same for `on: schedule`).

For the same commit, the Push workflow restore cache correctly while the CI workflow doesn't. That's the expected behavior based on the 3 conditions I'm using to disable cache (is a pr event, the `ci:no-cache` label is present and the event is on a "feature-branch" - not main nor stable/*) 
- https://github.com/Kerruba/poc-disable-cache-command/pull/5/checks?check_run_id=34603694007

When cache is enabled, both Push and CI workflows restore cache correctly
- https://github.com/Kerruba/poc-disable-cache-command/pull/5/checks?check_run_id=34604814824